test error on i386 in test_flip_two_edges

Hi Nico, does this error in `test_flip_two_edges` on i386 indicate there's a bug that needs to be fixed? If not, what's the best way to handle it??
```
_____________________________ test_flip_two_edges ______________________________

    def test_flip_two_edges():
        alpha = numpy.array([1.0, 3.0, 5.0, 7.0, 9.0, 11.0]) / 6.0 * numpy.pi
        R = [0.9, 1.0, 0.9, 1.0, 1.2, 1.0]
        points = numpy.array(
            [[r * numpy.cos(a), r * numpy.sin(a), 0.0] for a, r in zip(alpha, R)]
        )
        cells = numpy.array([[1, 3, 5], [0, 1, 5], [1, 2, 3], [3, 4, 5]])
        mesh = meshplex.MeshTri(points, cells)
        assert mesh.num_delaunay_violations() == 2
    https://ci.debian.net/data/autopkgtest/unstable/i386/p/python-meshplex/7605218/log.gz
        mesh.flip_until_delaunay()
        assert mesh.num_delaunay_violations() == 0
    
>       assert numpy.array_equal(
            mesh.cells["nodes"], [[5, 2, 3], [0, 2, 1], [5, 2, 0], [3, 4, 5]]
        )
E       assert False
E        +  where False = <function array_equal at 0xf6794778>(array([[2, 0, 1],\n       [2, 0, 5],\n       [5, 2, 3],\n       [3, 4, 5]]), [[5, 2, 3], [0, 2, 1], [5, 2, 0], [3, 4, 5]])
E        +    where <function array_equal at 0xf6794778> = numpy.array_equal

test/mesh_tri_test.py:613: AssertionError
```

Full test logs at
https://ci.debian.net/packages/p/python-meshplex/unstable/i386/
https://ci.debian.net/packages/p/python-meshplex/testing/i386/
https://ci.debian.net/data/autopkgtest/unstable/i386/p/python-meshplex/7605218/log.gz

--- a/test/mesh_tri/test_edge_flip.py
+++ b/test/mesh_tri/test_edge_flip.py
@@ -96,7 +96,9 @@ def test_flip_same_edge_twice():
 
 def test_flip_two_edges():
     alpha = numpy.array([1.0, 3.0, 5.0, 7.0, 9.0, 11.0]) / 6.0 * numpy.pi
-    R = [0.9, 1.0, 0.9, 1.0, 1.2, 1.0]
+    # Make the mesh slightly asymmetric to get the same flips on every architecture; see
+    # <https://github.com/nschloe/meshplex/issues/78>.
+    R = [0.95, 1.0, 0.9, 1.0, 1.2, 1.0]
     points = numpy.array(
         [[r * numpy.cos(a), r * numpy.sin(a), 0.0] for a, r in zip(alpha, R)]
     )
@@ -109,7 +111,7 @@ def test_flip_two_edges():
 
     mesh.show(show_point_numbers=True)
     assert numpy.array_equal(
-        mesh.cells["points"], [[5, 0, 2], [0, 1, 2], [5, 2, 3], [3, 4, 5]]
+        mesh.cells["points"], [[2, 5, 0], [2, 0, 1], [5, 2, 3], [3, 4, 5]]
     )
 
 
@@ -168,3 +170,7 @@ def test_flip_infinite():
     mesh = meshplex.MeshTri(points, cells)
     num_flips = mesh.flip_until_delaunay(tol=1.0e-13)
     assert num_flips == 0
+
+
+if __name__ == "__main__":
+    test_flip_two_edges()

--- a/test/mesh_tri/test_mesh_tri.py
+++ b/test/mesh_tri/test_mesh_tri.py
@@ -1,5 +1,6 @@
 import os
 import pathlib
+import platform
 import tempfile
 
 import meshio
@@ -23,14 +24,22 @@ def _compute_polygon_area(pts):
     )
 
 
-@pytest.mark.parametrize(
-    "cells_dtype",
-    [
-        numpy.int64
-        # depends on <https://github.com/numpy/numpy/issues/17760>
+# The dtype restriction is because of numpy.bincount.
+# See  <https://github.com/numpy/numpy/issues/17760> and
+# <https://github.com/nschloe/meshplex/issues/90>.
+cell_dtypes = []
+cell_dtypes += [
+    numpy.int32,
+    numpy.uint32,
+]
+if platform.architecture()[0] == "64bit":
+    cell_dtypes += [
+        numpy.int64,
         # numpy.uint64
-    ],
-)
+    ]
+
+
+@pytest.mark.parametrize("cells_dtype", cell_dtypes)
 def test_unit_triangle(cells_dtype):
     points = numpy.array([[0.0, 0.0], [1.0, 0.0], [0.0, 1.0]])
     cells = numpy.array([[0, 1, 2]], dtype=cells_dtype)

